.github/workflows/windows.yml

```diff
@@ -116,7 +116,8 @@ jobs:
       -
         name: Install Cypress
         run: cd etherpad && cd src && npm install cypress --legacy-peer-deps
-      - name: Install npm@6
+      - 
+        name: Install npm@6
         run: npm install npm@6 -g
```

Comment:
why install `npm` because it's already ship with node

Reply:
The problem is that Etherpad is one of the first projects that started to use node and npm. The only problem is that to run Etherpad you require this old npm version. More recent versions of npm will upgrade the package-lock.json file which breaks the Etherpad installation. There is a branch available that tries to do the npm update but there is unfortunately currently no one looking at it.

Reply:
I see. In short, the infrastructure is "old". We can start with typescript. 
I'm willing to open up an issue for the roadmap of how to modernise the infrastructure if you (the ether contributors) are interested.

Reply:
Yes that would be great. Thanks. I guess the best starting point is currently Typescript. After that current npm version and then move to ES instead of CJS.